File: 백준/Bronze/14761. FizzBuzz/FizzBuzz.cs
using System;
using System.IO;
using System.Text;

class Program
{
    static void Main(string[] args)
    {
        using (var sr = new StreamReader(Console.OpenStandardInput()))
        using (var sw = new StreamWriter(Console.OpenStandardOutput()))
        {
            string[] line = sr.ReadLine().Split();
            int X = int.Parse(line[0]);
            int Y = int.Parse(line[1]);
            int N = int.Parse(line[2]);

            for (int i = 1; i <= N; i++) 
            {
                if (i % X == 0 && i % Y == 0)
                    sw.WriteLine("FizzBuzz");
                else if (i % X == 0)
                    sw.WriteLine("Fizz");
                else if (i % Y == 0)
                    sw.WriteLine("Buzz");
                else
                    sw.WriteLine(i);
            }

            sw.Flush();
            sw.Close();
            sr.Close();
        }
    }
}

File: 백준/Bronze/28702. FizzBuzz/FizzBuzz.java
import java.io.*;

// BOJ 28702 - FizzBuzz
public class Main {

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

        int num = 0;
        boolean flag = false;

        for (int i = 0; i < 3; i++) {
            String line = br.readLine();
            char c = line.charAt(0);

            if (c != 'F' && c != 'B') {
                num = Integer.parseInt(line);
                flag = true;
            }

            if (flag)
                num++;
        }

        if (num % 3 == 0) {
            if (num % 5 == 0)
                bw.write("FizzBuzz\n");
            else
                bw.write("Fizz\n");
        } else {
            if (num % 5 == 0)
                bw.write("Buzz\n");
            else
                bw.write(num + "\n");
        }

        bw.flush();
        br.close();
        bw.close();
    }
}
